etna/analysis/forecast/plots.py

```diff
@@ -182,7 +189,7 @@ def plot_forecast(
                             segment_borders_df.index.values,
                             values_low,
                             values_high,
-                            facecolor=forecast_color,
+                            color=forecast_color,
                             alpha=alpha[interval_idx],
```

Comment:
Because of these alphas fillings become transparent and it is not seen.
We need to discuss how to calculate then

Reply:
What was the logic of calculating current `alpha`?